packages/nextjs/app/builders/0x7429CbD5eD20736645723E972bE60B7F6BF5959c/page.tsx

```diff
@@ -0,0 +1,158 @@
+    { icon: <Globe size={48} />, speed: 0.15, initialPos: { x: "34%", y: "78%" } },
+  ];
+
+  return (
+    <div className="relative min-h-screen overflow-hidden bg-gradient-to-br from-base-200 to-base-300">
+      {/* Parallax Background */}
+      <div className="absolute inset-0 z-0">
+        {parallaxLayers.map((layer, index) => (
+          <div
+            key={index}
+            className="absolute text-primary dark:text-base-100 opacity-40"
+            style={{
+              left: layer.initialPos.x,
+              top: layer.initialPos.y,
+              transform: `translate(
+                ${(mouseX - (typeof window !== "undefined" ? window.innerWidth : 0) / 2) * layer.speed * 0.01}px,
+                ${
+                  (mouseY - (typeof window !== "undefined" ? window.innerHeight : 0) / 2) * layer.speed * 0.01 -
+                  scrollY * layer.speed
+                }px
+              )`,
+              transition: "transform 0.3s ease-out",
+            }}
+          >
+            {layer.icon}
+          </div>
+        ))}
+      </div>
+
+      {/* Floating Particles */}
+      {[...Array(20)].map((_, i) => (
+        <div
+          key={i}
+          className="absolute rounded-full bg-primary dark:bg-base-200 opacity-30"
+          style={{
+            width: Math.random() * 20 + 10 + "px",
+            height: Math.random() * 20 + 10 + "px",
+            left: Math.random() * 100 + "%",
+            top: Math.random() * 100 + "%",
+            transform: `translateY(${scrollY * (Math.random() * 0.2 + 0.05)}px)`,
+            transition: "transform 2s ease-out",
+          }}
+        />
+      ))}
+
+      {/* Main Content */}
+      <article className="relative z-10 flex items-center justify-center min-h-screen p-4">
+        <div className="bg-base-100/80 backdrop-blur-md shadow-lg hover:shadow-xl transition-shadow duration-300 rounded-lg w-full max-w-2xl">
+          <div className="p-6">
+            <div className="flex items-center space-x-4">
+              <Image
+                src="/assets/dvm0x742_avatar.jpg"
+                width={100}
+                height={100}
+                alt="Profile"
+                className="rounded-full"
+              />
+              <div>
+                <h1 className="text-2xl font-bold text-base-content">Michael Ojekunle</h1>
+                <p className="text-secondary-content flex flex-col gap-1">
+                  <span>Frontend Developer / Smart Contract Writer</span>
+                  <span className="flex items-center text-sm opacity-70">
+                    0x7429CbD5eD20736645723E972bE60B7F6BF5959c
+                    {copied ? (
+                      <Check className="w-3 h-3 ml-2 text-green-500 transition" />
+                    ) : (
+                      <Copy className="w-3 h-3 ml-2 cursor-pointer transition" onClick={handleCopy} />
+                    )}
+                  </span>
```

Comment:
You might consider using the SE2 component Address to leverage the existing component. There you already have the copy functionality.